sql.txt

```diff
@@ -1,15 +1,24 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+select CustomerName, ContactName, Country FROM Customers order by country limit 10 offset 10;
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
+select * from Customers where ContactName like "A%";
 
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
+select * from OrderDetails where ProductID = 51 and quantity > 10;
 
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+update Products values 
+Insert into Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price) values (78, 'a', 100, 100, "100boxes", 100);
+Insert into Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price) values (78, 'b', 100, 100, "1000boxes", 100);
+Insert into Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price) values (78, 'c', 100, 100, "10000boxes", 100);
 
 (5) Update the two top rows of the Products Table to increase the price by 1.50. (Just specify the new value; do 
 not try to get SQL to do the addition for you.)  Paste your SQL statement below
+Update Products set Price = Price + 1.50 Where ProductID = 1 or ProductID = 2;
```

Comment:
We have to add the 1.5 manually, not using the addition operator.

Reply:
Would "UPDATE Products SET Price = 22.5 WHERE ProductID = 1; UPDATE Products SET Price = 23.5 WHERE ProductID = 2;" work?

Reply:
It looks good. By the way, I think the initial values are 18 and 19 for ProductID 1 and 2.